neo/Consensus/ConsensusContext.cs

```diff
@@ -46,6 +47,8 @@ internal class ConsensusContext : IConsensusContext
         private readonly Wallet wallet;
         private readonly Store store;
 
+        public StateRoot StateRoot { get; set; }
```

Comment:
What it's the difference of StateRoot and Root?

Reply:
I think `Root` is a proposal `StateRoot`, the `StateRoot` variable is a complete state root with signatures.